### ISSUE TITLE
Open code location of test not working

### ISSUE THREAD

**Issue by j-o-d-o** [csoltenborn/GoogleTestAdapter#216]
This key feature is currently not working for me. (VS 2015 Version 14.0 Update 3).
It just does not do anything. The summary window says: "Source: no source available". But running the tests as well as the discovery is working perfectly fine.

**Comment by csoltenborn** [comment]
Have you checked out the trouble shooting section of the docs? It contains a [whole subsection](https://github.com/csoltenborn/GoogleTestAdapter#no-source-locations-and-traits-are-found-for-my-tests) on missing source locations... Let me know whether this helped.

**Comment by j-o-d-o** [comment]
Yes, I have checked that section (granted, I can't find any _Parse symbol information_ setting). 

A potential issue could be, that my project is in a totally different location as the solution and is included into the solution via "add existing project". Maybe that way GTA can not find the .pdb file which is created at the location of the solution file.

**Comment by csoltenborn** [comment]
Did you enable debug mode? It will tell you more about what's going on... In particular, it will tell you where it looked for the pdb.

The *Parse symbol information* is located at Options -> Google Test Adapter -> General.

**Comment by j-o-d-o** [comment]
Okay, I overlooked that there actually are settings for the Google Test Adapter in the options... 
It seems like the .pdb file is found and can be parsed for test discovery but no source location is found.
Thats the output:
```
Solution settings: AdditionalTestExecutionParam: '', BatchForTestSetup: '', BatchForTestTeardown: '', BreakOnFailure: False, CatchExceptions: True, DebugMode: True, KillProcessesOnCancel: False, MaxNrOfThreads: 8, NrOfTestRepetitions: 1, ParallelTestExecution: False, ParseSymbolInformation: True, PathExtension: '', PrintTestOutput: False, RunDisabledTests: False, ShowReleaseNotes: True, ShuffleTests: False, ShuffleTestsSeed: 0, TestDiscoveryRegex: '', TestDiscoveryTimeoutInSeconds: 30, TestNameSeparator: '', TimestampOutput: False, TraitsRegexesAfter: {}, TraitsRegexesBefore: {}, UseNewTestExecutionFramework: True, WorkingDir: '$(ExecutableDir)'
No settings configured for test executable 'MY_PATH\Debug\tp_test.exe'; running with solution settings: AdditionalTestExecutionParam: '', BatchForTestSetup: '', BatchForTestTeardown: '', BreakOnFailure: False, CatchExceptions: True, DebugMode: True, KillProcessesOnCancel: False, MaxNrOfThreads: 8, NrOfTestRepetitions: 1, ParallelTestExecution: False, ParseSymbolInformation: True, PathExtension: '', PrintTestOutput: False, RunDisabledTests: False, ShowReleaseNotes: True, ShuffleTests: False, ShuffleTestsSeed: 0, TestDiscoveryRegex: '', TestDiscoveryTimeoutInSeconds: 30, TestNameSeparator: '', TimestampOutput: False, TraitsRegexesAfter: {}, TraitsRegexesBefore: {}, UseNewTestExecutionFramework: True, WorkingDir: '$(ExecutableDir)'
Google Test indicators found in executable MY_PATH\Debug\tp_test.exe
Parsing pdb file "MY_PATH\Debug\tp_test.pdb"
Found 1 test method symbols and 0 trait symbols in binary MY_PATH\Debug\tp_test.exe
Warning: Could not find source location for test example_case.example_test
Added testcase example_case.example_test.example_test
Found 1 tests in executable MY_PATH\Debug\tp_test.exe
```

I do have _Parse Symbol Information_ set to True as well as _Properties -> Linker Debugging -> Generate Debug Info_ set to Yes (/DEBUG).

**Comment by csoltenborn** [comment]
Looks like I need the solution (preferably a stripped-down version) to be able to help you... Any chance that you can provide it?

Btw: There was a minor issue with test cases (see #209) which has been fixed with the soon-to-be-released version 0.13.0. I don't think that this is necessarily related to your problem, but you could give [this build](https://ci.appveyor.com/project/csoltenborn/googletestadapter-u1cxh/build/0.13.0.1071/artifacts) a chance anyways...

**Comment by csoltenborn** [comment]
Having a brief look at your log output again, I noticed this kind of weird test name: `example_case.example_test.example_test`. This is surprising, especially considering the line above, where it looks for the source location of test `example_case.example_test`.

My guess is that either you are using some invalid naming scheme, or that the GTA test name parser can not deal with that notion of test names. Can you provide the code producing that particular test?

**Comment by j-o-d-o** [comment]
I think I did some copying error there when I added the MY_PATH placeholder. Both times it says `example_case.example_test` when looking at the output again. Sorry for the confusion. 

I will try the new build and try to reproduce it in a basic solution. Thanks for the quick responses!

**Comment by j-o-d-o** [comment]
Okay, while trying to reproduce this error I found the source of the issue.
I have to change the google test sources in a way to have this interface
`TEST(example_case, "example_test")` 
In order to make our official reporting tool happy... :/

If I use the original google test interface, it is working just fine. Thus, the issue is not within the Google Test Adapter but my own code. 
Thanks for all the help and the great tool. Closing this.
